src/Query/Filter/UrgencyField.ts

```diff
@@ -58,4 +58,16 @@ export class UrgencyField extends Field {
             return [`${task.urgency.toFixed(2)}`];
         };
     }
+
+    /**
+     * The {@link Field.createGrouper} creates a grouper that sorts by increasing values.
+     * For {@link UrgencyField} the group sorting shall be done by decreasing values, so
+     * the normal order here is the reverse of the regular one.
+     *
+     * @param reverse - false for normal group order (from most urgent to less urgent),
+     * true for reverse group order.
+     */
+    public createGrouper(reverse: boolean): Grouper {
+        return super.createGrouper(!reverse);
```

Comment:
I decided to go for this implementation rather than
```js
    return new Grouper(this.fieldNameSingular(), this.grouper(), !reverse);
```
as I wanted to show that the order is connected to the one, but is reversed. I think this makes the code more evident rather than spotting the `!` in one of the parameters.

Feel free to change if you think otherwise =)

Reply:
Yes, that's really nice. A good spot. Thank you.